src/actions/classifier.js

```diff
@@ -25,6 +25,11 @@ export function startNewClassification(workflowID) {
     }).then(() => {
       return dispatch(setNeedsTutorial())
     }).then(() => {
+      if (getState().classifier.guide[workflowID] !== undefined) {
+        return
+      }
+      return dispatch(fetchFieldGuide(workflowID))
+    }).then(() => {
```

Comment:
Oh. I was thinking, for example, that lines 22, 24, and 26 could be in the same `.then()` block, but if they are async/dependent on each other and need to be chained, that's fine.